tests/acceptance/tour-test.js

```diff
@@ -36,38 +41,14 @@ describe('Acceptance | tour', function() {
 
     await visit('/')
 
-    expect(find('.modal--visible')).to.not.be.ok
-  })
-
-  it('does not show a welcome dialog when later clicked', async function() {
-    await visit('/')
-
-    expect(find('.modal--visible')).to.exist
-
-    await click('.btn-default')
-
-    await visit('/someotherroute')
-    await visit('/')
-
-    expect(find('.modal--visible')).to.not.exist
-  })
-
-  it('can ignore tour permanently', async function() {
-    await visit('/')
-
-    await click('.btn-never')
-
-    await visit('/someotherroute')
-    await visit('/')
-
-    expect(find('.modal--visible')).to.not.be.ok
```

Comment:
Why remove those tests?

Reply:
The used addon ember-site-tour changed and removed the later button.